lib/server.js

```diff
@@ -62,6 +61,9 @@ const listener = (req, res) => {
     }
     receiveBody(req).then(data => {
       client.message(data);
+    }, err => {
+      application.logger.error(err.stack);
+      client.error(500);
```

Comment:
This will write logs: 
```suggestion
      client.error(500, err);
```